src/requests/getProperty.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+
+const getProperty = (setProperties, setAlert, query) => {
+  let endpoint = "http://localhost:4000/api/v1/PropertyListing";
+
+  if (query) {
+    endpoint += query;
+  }
+
+  return axios
+    .get(endpoint)
+    .then((response) => {
+      setProperties(response.data);
+      if (response.data.length === 0) {
+        setAlert({ message: "No property in this city, please try again!" });
```

Comment:
This is the same as Ive commented in getSavedProperties, message and isSuccess are both required.